geotrek/api/v2/serializers.py

```diff
@@ -1406,15 +1406,28 @@ class Meta:
             model = signage_models.Blade
             fields = ('id', 'number', 'color', 'direction', 'lines')
 
+    class SignageConditionSerializer(DynamicFieldsMixin, serializers.ModelSerializer):
+
+        class Meta:
+            model = signage_models.SignageCondition
+            fields = ('id', 'label', 'structure')
+
     class SignageSerializer(DynamicFieldsMixin, serializers.ModelSerializer):
         geometry = geo_serializers.GeometryField(read_only=True, source="geom3d_transformed", precision=7)
         structure = serializers.CharField(source='structure.name')
         attachments = AttachmentSerializer(many=True)
         blades = BladeSerializer(source='blades_set', many=True)
+        condition = serializers.SerializerMethodField(
```

Comment:
What is the purpose of exposing the condition through the API? I am not sure other maintenance attributes are API-exposed. (I just checked: signages' sealings are exposed -- a previous mistake?)